low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -103,15 +107,22 @@ def initialize(self, conf_reader, msgQlist, product):
         self._rack_id = self._conf_reader._get_value_with_default(
                                 self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.RACK_ID), '001')
         self._node_id = self._conf_reader._get_value_with_default(
-                                self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.NODE_ID), '001')  
+                                self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.NODE_ID), '001')
         self._timestamp_file_path = self._conf_reader._get_value_with_default(
                                     self.RAIDIntegritySensor, self.TIMESTAMP_FILE_PATH_KEY, self.DEFAULT_TIMESTAMP_FILE_PATH)
         self._polling_interval = int(self._conf_reader._get_value_with_default(
                                 self.RAIDIntegritySensor, self.POLLING_INTERVAL, self.DEFAULT_POLLING_INTERVAL))
+        self._next_scheduled_time = self._polling_interval
+        self._fault_accepted_time = int(self._conf_reader._get_value_with_default(
+                                self.RAIDIntegritySensor, self.FAULT_ACCEPTED_TIME, self.DEFAULT_FAULT_ACCEPTED_TIME))
+
+        if self._polling_interval < self.MIN_POLLING_INTERVAL:
+            self._polling_interval = self.MIN_POLLING_INTERVAL
+
         self.utility = Utility()
         if self.utility.is_env_vm():
             self.shutdown()
```

Comment:
I think this can be removed, since `sspl_constansts.py` already has a list
`DISABLE_FOR_VIRTUAL_SERVER = ("CPUFaultSensor", "RAIDIntegritySensor",  "RAIDsensor", "MemFaultSensor", "NodeHWsensor")`
which prevents these sensors to run In virtual environemnt.

Reply:
no, check similar previous comment

Reply:
sure.